### ISSUE TITLE
Documentation on coloring the themes

### ISSUE THREAD

**Issue by CheeseCrustery** [Bash-it/bash-it#1864]
I am looking at the [powerline theme](https://github.com/Bash-it/bash-it/blob/master/themes/powerline/powerline.theme.bash) right now. I know I can configure it's colors, for example by doing `export USER_INFO_THEME_PROMPT_COLOR={some integer value}`. I have tried out quite a few values, but the outcome is seemingly random - I can't for the life of me figure out how they're encoded. The [official documentation](https://bash-it.readthedocs.io/en/latest/themes-list/powerline-base/) does not even mention coloring.

Please explain, I'm getting desperate.

**Comment by davidpfarrell** [comment]
Greetings!

As I recall powerline supports standard 256 terminal colors.

What I think you'll want is a way to review which what colors each value renders in your terminal.
I googled `"print 256 colors terminal"` and found this page:

* https://askubuntu.com/questions/821157/print-a-256-color-test-pattern-in-the-terminal

The answer I went with uses this gist:
* [HaleTom/print256colours.sh](https://gist.github.com/HaleTom/89ffe32783f89f403bba96bd7bcd1263)

Here's what the script looks like in my terminal:
<img width="995" alt="Screen Shot 2021-04-05 at 10 27 59 AM" src="https://user-images.githubusercontent.com/45741407/113604138-b5894680-95f9-11eb-9412-2cb55ba45b00.png">

*NOTE: I have modified values 0-15 to represent the [Solarized Color Scheme](https://ethanschoonover.com/solarized/)

You *should* find that the color/numbers correspond to those used in the powerline themes.

Hope that helps - lemme know how it goes!
